integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/strategy/outbox/OutboxPollingOutOfOrderIdsIT.java

```diff
@@ -358,6 +366,16 @@ private void updateOutboxTableRow(Session session, Integer oldId, Integer newId)
 					env.getDialect() instanceof H2Dialect
 			);
```

Comment:
Hu, I didn't know that. Any chance we can actually run this test on other databases? As we've seen, correct ordering depends a lot on the underlying database's capabilities, so we better test every database we can.

Reply:
You're right, so that we could test make this running for JDK8 profile.